models/correlomatrix_detector.cpp

```diff
@@ -349,7 +349,7 @@ nest::correlomatrix_detector::handle( SpikeEvent& e )
       for ( SpikelistType::const_iterator spike_j = otherSpikes.begin(); spike_j != otherSpikes.end(); ++spike_j )
       {
         size_t bin;
-        long other = spike_j->receptor_channel_;
+        decltype( sender ) other = spike_j->receptor_channel_;
         long sender_ind, other_ind;
```

Comment:
```suggestion
        decltype( sender ) other = spike_j->receptor_channel_;
        decltype( sender ) sender_ind = 0;
        decltype( sender ) other_ind = 0;
```

Reply:
@muffgaga Could you accept my suggestion and merge master into the branch for this PR?

Reply:
Sorry, for the delay. Yes.